Parser/spark.py

```diff
@@ -644,7 +645,7 @@ def buildTree(self, nt, item, tokens, k):
                 attr[i] = self.buildTree(sym, why[0],
                                          tokens, why[1])
                 item, k = self.predecessor(key, why)
-        return self.rule2func[self.new2old[rule]](attr)
+        return self.rule2func[self.new2old[rule]](*attr)
```

Comment:
Shouldn't the call in `deriveEpsilon()` be updated too?

Reply:
I think so yes. Fixed in e14e5361932c181b1c0ab9be7b685fdfbc23f8a8.